### ISSUE TITLE
the output dimension of the network

### ISSUE THREAD

**Issue by breadbread1984** [Illumina/PrimateAI#1]
I read your keras code and find that the output dimension of the network is batchx20(length)x40(channel). is there a full connection layer missing before the crossentropy loss is calculated? if not how do you classify with tensor of dimension batch x 20 x 40 directly?
